app/src/main/java/com/xinto/opencord/domain/model/Message.kt

```diff
@@ -26,7 +26,9 @@ data class DomainMessageRegular(
     override val author: DomainUser,
     val editedTimestamp: Instant?,
     val attachments: List<DomainAttachment>,
-    val embeds: List<DomainEmbed>
+    val embeds: List<DomainEmbed>,
+    val isReply: Boolean,
+    val referencedMessage: DomainMessageRegular?
```

Comment:
`referencedMessage` should probably be cached manually in addition with this message

Reply:
Also related to CacheManager